Core/src/main/java/com/github/intellectualsites/plotsquared/plot/object/Plot.java

```diff
@@ -3076,18 +3070,18 @@ public boolean move(final Plot destination, final Runnable whenDone, boolean all
         final int offsetZ = db.getZ() - ob.getZ();
         if (this.owner == null) {
             TaskManager.runTaskLater(whenDone, 1);
-            return false;
+            return CompletableFuture.completedFuture(false);
         }
-        boolean occupied = false;
+        AtomicBoolean occupied = new AtomicBoolean(false);
         Set<Plot> plots = this.getConnectedPlots();
```

Comment:
Why are we swapping connected plots if we're not allowing moving/swapping of connected plots?

Reply:
I didn't really want to remove the code, although it can be removed without consequences so I guess it'd be a good idea to.

Reply:
But also, there's no real harm in allowing the code to swap merged plots. We just disallow it in the command as it's not something we really want to support, kind of.